packages/zapp/console/src/components/LaunchPlan/SearchableLaunchPlanNameList.tsx

```diff
@@ -0,0 +1,167 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import DeviceHub from '@material-ui/icons/DeviceHub';
+import classNames from 'classnames';
+import { useNamedEntityListStyles } from 'components/common/SearchableNamedEntityList';
+import { useCommonStyles } from 'components/common/styles';
+import { separatorColor, primaryTextColor, launchPlanLabelColor } from 'components/Theme/constants';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { Routes } from 'routes/routes';
+import { debounce } from 'lodash';
+import { Typography, FormGroup } from '@material-ui/core';
+import { LaunchPlanListStructureItem } from './types';
+import { SearchableInput } from '../common/SearchableList';
+import { useSearchableListState } from '../common/useSearchableListState';
+
+interface SearchableLaunchPlanNameItemProps {
+  item: LaunchPlanListStructureItem;
+}
+
+interface SearchableLaunchPlanNameListProps {
+  launchPlans: LaunchPlanListStructureItem[];
+}
+
+export const showOnHoverClass = 'showOnHover';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  container: {
+    padding: theme.spacing(2),
+    paddingRight: theme.spacing(5),
+  },
+  filterGroup: {
+    display: 'flex',
+    flexWrap: 'nowrap',
+    flexDirection: 'row',
+    margin: theme.spacing(4, 5, 2, 2),
+  },
+  itemContainer: {
+    marginBottom: 15,
+    borderRadius: 16,
+    padding: '23px 30px',
```

Comment:
Can you use theme.spacing() here too?
You can use float values for spacing, 0.5 and 0.25 steps are acceptable. in here is 24 px instead of 23 make a big differnce? If yes - feel free to leave as is.

Reply:
fixed